test-case/check-alsabat.sh

```diff
@@ -64,11 +72,52 @@ function __upload_wav_file
     done
 }
 
+function set_pga_to_unity_gain
+{
+    tmp=$(amixer controls | grep "$pga" | grep Volume)
+    search="name="
+    cname=${tmp#*$search}
+
+    # Get volume min and step to compute value for cset
+    # for 0 dB gain. The amixer line looks like
+    # "| dBscale-min=-50.00dB,step=1.00dB,mute=1"
+    scale=$(amixer cget name="$cname" | grep "dBscale")
+    search="dBscale-min="
+    tmp=${scale#*$search}
+    min_db="${tmp%%dB*}"
+    search="step="
+    tmp=${scale#*$search}
+    step_db="${tmp%%dB*}"
+
+    # Get multiplied by 100 values by removing decimal dot
+    min_x100="${min_db//.}"
+    step_x100="${step_db//.}"
+    val=$(printf %d "$(((-$min_x100) / $step_x100))")
```

Comment:
Nit: you don't need the `'$'` inside `$(( ))`

Reply:
Thanks, I didn't know nor understand what shellcheck suggested!